hip/static/js/index.js

```diff
@@ -24,6 +24,8 @@ import SideBar from "../../../apps/hip/static/js/sidebar";
 import RightScrollNav from "../../../apps/hip/static/js/rightScrollNav";
 import MobileSearchModal from "../../../apps/hip/static/js/mobileSearchModal";
 import HealthAlertFilter from "../../../apps/health_alerts/static/js/healthAlertFilter";
+import HealthAlertSignup from "../../../apps/health_alerts/static/js/healthAlertSignup";
+import healthAlertMessage from "../../../apps/health_alerts/static/js/healthAlertMessage";
```

Comment:
Curious if there is a reason this one starts with a lowercase, unlike the others?

Reply:
Nope, just was rushing to get the thought and didn't correct spelling to fit the convention. Will do.

Reply:
done